jdbc-core/src/main/java/com/salesforce/datacloud/jdbc/HyperDatasource.java

```diff
@@ -81,12 +83,16 @@ public static DataCloudConnection connectUsingProperties(@NonNull String url, Pr
             int port = jdbcUrl.getPort();
             val properties = info != null ? (Properties) info.clone() : new Properties();
             jdbcUrl.addParametersToProperties(properties);
+
+            // Always use SSL - let SslProperties determine the mode
+            SslProperties sslProps = SslProperties.ofDestructive(properties);
+
             val connectionProperties = ConnectionProperties.ofDestructive(properties);
             val grpcChannelProperties = GrpcChannelProperties.ofDestructive(properties);
             PropertyParsingUtils.validateRemainingProperties(properties);
```

Comment:
Let's stay consistent with the surrounding code, i.e.:
* use `val`
* name it `Properties` not `Props`

```suggestion
            val sslProperties = SslProperties.ofDestructive(properties);
            val connectionProperties = ConnectionProperties.ofDestructive(properties);
            val grpcChannelProperties = GrpcChannelProperties.ofDestructive(properties);
            PropertyParsingUtils.validateRemainingProperties(properties);
```